### PR TITLE
Add support for SQS timeouts

### PR DESCRIPTION
**Pull request by mtgray-sp**

### CODE REVIEW

**Review by lucas-mitri-messagebird — APPROVED** [review]
LGTM. I was looking at this code while you worked on it too.

The tests are passing for me.

For other services that won't use these options, the only change they'll see is that instead of the property `httpOptions` being missing, it'll be `{}`, which shouldn't affect anything.

File: lib/sqs/index.js [hunk 1]
```diff
@@ -36,13 +36,30 @@ module.exports = ({
...
+    extraHttpOptions.timeout = sqsTimeout;
+  }
+  if (sqsConnectTimeout) {
+    extraHttpOptions.connectTimeout = sqsConnectTimeout;
+  }
+  const SQS = setHttpAgent(
+    {
+      ...clientConfig,
+      ...ep,
+      httpOptions: { ...clientConfig.httpOptions, ...extraHttpOptions }
```
Comment (NupurKulkarni1) [inline lib/sqs/index.js:58]: what happens when client config has http options and we also pass sqsTimeout and sqsConnectTimeout simultaneously?

**Review by NupurKulkarni1 — COMMENTED** [review]
I just left a small comment but otherwise lgtm!

**Review by NupurKulkarni1 — APPROVED** [review]

File: lib/sqs/index.js (hunk 1 above)
Comment (mtgray-sp) [reply lib/sqs/index.js:58]: We will merge the client config's http options with the extra http options

**Pull request merged**